app/controllers/prizes_controller.rb

```diff
@@ -23,6 +23,6 @@ def win_prize
   private
 
   def prize_params
-    params.require(:prize).permit(:name, :percentage, :image, :is_merch)
+    params.require(:prize).permit(:name, :percentage, :image, :prize_type, :description, :identifier)
```

Comment:
Shouldn't this be `type` now?

Reply:
Was getting an error and figured that type is a reserved keyword I believe, used `prize_type` instead in this case.

Reply:
And it's okay to have `type` in the migration?

Reply:
Haven't done Rails for a while. 😅 Did you have to roll back? And create a new migration?